src/api-umbrella/proxy/startup/seed_database.lua

```diff
@@ -160,6 +175,15 @@ local function seed_initial_superusers()
       data["authentication_token"] = random_token(40)
       data["created_at"] = nowMongoDate
       data["updated_at"] = nowMongoDate
+      
+      -- Changes added by FIWARE Foundation (Chandra) for setting initial super users - Start
+      ngx.log(ngx.INFO, "Chandra user name is email -- Printing the value in if->notAdmin loop : ", emailUserName)
+      if emailUserName then
```

Comment:
In case username and email are configured differently, wouldn't it be better to throw an error instead of a silent replace?